README.md

```diff
@@ -1,3 +1,5 @@
+[![Puppet Forge](http://img.shields.io/puppetforge/v/katello/gutterball.svg)](https://forge.puppetlabs.com/katello/gutterball)
```

Comment:
Isn't there a https image available? It isn't showing on github now, which I think is the most important one.